bot/cogs/bot.py

```diff
@@ -60,6 +62,27 @@ def __init__(self, bot: AutoShardedBot):
 
         await ctx.invoke(self.info)
 
+    async def on_message(self, msg: Message):
+        if msg.content.count("\n") >= 3:  # more than three lines
+            try:
+                tree = ast.parse(msg.content)
+                # Attempts to parse the message into an AST node.
+                # Invalid Python code will raise a SyntaxError.
+                if not all(isinstance(node, ast.Expr) for node in tree.body):
+                    # we dont want multiple lines of single words like:
+                    '''
+                    hi
+                    there
+                    guys
+                    '''
+                    # that would be syntactically valid Python but in this case
+                    # just some random multiline text someone is sending.
+
+                    await msg.channel.send("```python\n{}\n```".format(msg.content))
```

Comment:
If msg.content were >1986 characters long, discord will get angry at you as the message exceeds 2000 characters, to avoid unnecessary errors I suggest some sort of check against this? Maybe format the string, then check if it is over 2000 characters before sending and deleting